src/Connection/Parameters.php

```diff
@@ -25,6 +26,9 @@ class Parameters implements ParametersInterface
         'scheme' => 'tcp',
         'host' => '127.0.0.1',
         'port' => 6379,
+        'set_client_info' => true,
+        'client_name' => 'predis',
+        'client_version' => Client::VERSION,
```

Comment:
Should these be configurable @vladvildanov?

Reply:
@tillkruss It is configurable, check [there](https://github.com/predis/predis/pull/1370/files#diff-7171e490ba29ca3a384a9f5d48189766b2f8c9f8af0371d523c8482770d65a0eR36). In this place we set a default values

Reply:
I mean should it be? Why wouldn't it always be Predis, if it's enabled?

Reply:
As for what I'm concerned, we just need a way to opt-out to solve the performance issue. IMO [Added overriding for client name and version, increased test coverage](https://github.com/predis/predis/pull/1370/commits/1f6c5525e1d47f7b2167d26419f5f2365dc2775f) is another feature, this could be submitted and discussed in a separated merge-request.

Reply:
Yeah, let's go with only `set_client_info` for now, so we can release it. 

I don't see the need for name/version configuration.

Reply:
https://github.com/predis/predis/pull/1370#issuecomment-1697735969